### PR TITLE
New left-hand TOC

### PR DESCRIPTION
**Pull request by beckykd**
### Details and comments
Fixes #

### CODE REVIEW

File: docs/how_to/error-mitigation.rst [hunk 1]
```diff
@@ -0,0 +1,232 @@
...
+    psi1_H1 = job.result()  
+
+.. note::
+    As you increase the resilience level, you will be able to leverage additional methods to improve the accuracy of your result. However, because the methods become more advanced with each level, they require additional sampling overhead (time) to generate more accurate expectation values.     
+
+Configure Sampler with resilience levels 
+-----------------------------------------
+
+
+The Sampler default resilience setting (level 1) does not enable error mitigation to allow users to generate unmitigated probability distributions. Users can enable one resilience level for sampling tasks, which allows them to leverage readout error mitigation as described below.
```
Comment (jyu00) [inline docs/how_to/error-mitigation.rst:147]: > The Sampler default resilience setting (level 1) does not enable error mitigation

Level 1 _does_ enable error mitigation? I think that's residual from when default was level 0.

File: docs/how_to/error-suppression.rst [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+
+    from qiskit_ibm_runtime import QiskitRuntimeService, Session, Estimator, Options
+
+    service = QiskitRuntimeService()
+    options = Options(optimization_level=2)
+
+    with Session(service=service, backend="ibmq_qasm_simulator") as session:
+    estimator = Estimator(session=session, options=options)
+    job = estimator.run(circuits=[psi1], observables=[H1], parameter_values=[theta1])
+    psi1_H1 = job.result()
```
Comment (jyu00) [inline docs/how_to/error-suppression.rst:62]: Missing indentation

File: docs/how_to/error-suppression.rst [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+
+.. code-block:: python
+
+    from qiskit_ibm_runtime import QiskitRuntimeService, Session, Sampler, Options
+
+    service = QiskitRuntimeService()
+    options = Options(optimization_level=3)
+
+    with Session(service=service, backend="ibmq_qasm_simulator") as session:
+    sampler = Sampler(session=session, options=options)
```
Comment (jyu00) [inline docs/how_to/error-suppression.rst:78]: Same as above; needs indentation

File: docs/how_to/error-suppression.rst [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+  
+Advanced transpilation options
+------------------------------
+
+You also have the ability to tune a variety of advanced options to configure your transpilation strategy further. These methods can be used alongside optimization levels.  They allow you to change the options of interest and let your optimization level manage the rest.  
+
+Most of the transpilation options are inherited from `qiskit.compiler.transpile <https://qiskit.org/documentation/stubs/qiskit.compiler.transpile.html>`__.
```
Comment (jyu00) [inline docs/how_to/error-suppression.rst:85]: I think you have an extra `_` in the end

File: docs/cloud/quickstart.rst [hunk 5]
```diff
@@ -0,0 +1,140 @@
...
+
+   from qiskit_ibm_runtime import QiskitRuntimeService
+
+   # Save account to disk.
+   QiskitRuntimeService.save_account(channel="ibm_cloud", token="<IBM Cloud API key>", instance="<IBM Cloud CRN>")
+
+   service = QiskitRuntimeService()
+
+
+If you need to update your saved credentials, run ``save_account`` again, passing in ``overwrite=True``  and the updated credentials. For more information about managing your account, see the `account management tutorial <https://qiskit.org/documentation/partners/qiskit_ibm_runtime/tutorials/04_account_management.html>`__.
```
Comment (jyu00) [inline docs/cloud/quickstart.rst:97]: Is there going to be an `account management tutorial`? The new TOC doesn't have it

File: docs/cloud/quickstart.rst [hunk 6]
```diff
@@ -0,0 +1,140 @@
...
+
+
+Choose a program to run
+----------------------------------
+
+
+Qiskit Runtime uses primitive programs to interface with quantum computers. The following programs are publicly available. Choose the appropriate link to continue learning how to run a program.
+
+`Getting started with Sampler <https://qiskit.org/documentation/partners/qiskit_ibm_runtime/tutorials/how-to-getting-started-with-sampler.html>`__
+`Getting started with Estimator <https://qiskit.org/documentation/partners/qiskit_ibm_runtime/tutorials/how-to-getting-started-with-estimator.html>`__
```
Comment (jyu00) [inline docs/cloud/quickstart.rst:140]: The rendered page has both links on the same line, which is hard to read. I think an empty line between them would fix it.

File: docs/cloud/quickstart-org.rst [hunk 7]
```diff
@@ -0,0 +1,117 @@
...
+To capture App ID events, open your App ID instance, then navigate to **Manage Authentication -> Authentication settings** and enable **Runtime Activity**.
+
+Define more fine grained roles
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The actions in the custom roles can be used for more fine grained access control. For example, some users might need full access to work on service instances, while others might only need Read access to service instances, programs, and jobs.
+
+To achieve that, define two different custom roles such as ``MLreader`` and ``MLwriter``. Remove all cancel, delete, and update roles from the ``MLreader`` custom role, and include all actions in the ``MLwriter`` custom role. Next, add the roles to two different access groups accordingly.
+
+When using dynamic rules, that is, when the IDP administrator manages access through custom IDP user attributes, do not use IDP custom user attributes that are substrings of each other. For instance, don’t use ``ml`` and ``mlReader``, as the string comparison of ``ml`` would also accept ``mlReader``. You could use ``MLreader`` and ``MLwriter`` to avoid this conflict. {: note}
```
Comment (jyu00) [inline docs/cloud/quickstart-org.rst:88]: I think the `{: note}` in the end is a mistake?

File: docs/cloud/quickstart-steps-org.rst [hunk 8]
```diff
@@ -0,0 +1,112 @@
...
+   6. Repeat this step if you want to give an access group permission to several service instances.
+
+Set up your ID provider and assign users
+----------------------------------------
+
+Follow the steps in the appropriate topic, depending on the ID provider you chose to use:
+
+-  `Use IBM Cloud as the ID provider for users that have IBM Cloud accounts <cloud-provider-org>`__
+-  `Use IBM Cloud as the ID provider for users that do not have IBM Cloud accounts <appid-cloud-org>`__
+-  `Use an ID provider other than IBM Cloud for users that do not have IBM Cloud accounts <appid-org>`__
```
Comment (jyu00) [inline docs/cloud/quickstart-steps-org.rst:105]: These links don't work for me

File: docs/cloud/data-security.rst [hunk 9]
```diff
@@ -0,0 +1,21 @@
...
+Protecting your sensitive data in Qiskit Runtime
+------------------------------------------------
+
+The data that you store in IBM Cloud is encrypted at rest by using a randomly generated key.
+
+
+Deleting your data in Qiskit Runtime
+------------------------------------
+
+Deleting a service instance removes all of the content associated with that instance, such as your jobs, results, parameters, and programs. To delete an instance, from the [Instances page](https://cloud.ibm.com/quantum/instances), find the instance you want to remove, click its overflow menu, then click **Delete**. You will be asked to confirm the deletion.
```
Comment (jyu00) [inline docs/cloud/data-security.rst:16]: The `Instance page` link doesn't render correctly for some reason:

image

maybe needs a space before the comma?

File: docs/cloud/data-security.rst [hunk 10]
```diff
@@ -0,0 +1,21 @@
...
+
+Deleting Qiskit Runtime instances
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The Qiskit Runtime data retention policy describes how long your data is stored after you delete the service. The data retention policy is included in the Qiskit Runtime service description, which you can find in the [{{site.data.keyword.cloud_notm}} Terms](https://www.ibm.com/support/customer/csol/terms?id=i126-9425&lc=en#detail-document).
```
Comment (jyu00) [inline docs/cloud/data-security.rst:21]: The link doesn't render correctly either

File: docs/cloud/at-events.rst [hunk 11]
```diff
@@ -0,0 +1,38 @@
+Audit events for Qiskit Runtime
+===============================
+
+As a security officer, auditor, or manager, you can use the IBM Cloud® Activity Tracker service to track how users and applications interact with the Qiskit Runtime service in IBM Cloud.
+
+IBM Cloud Activity Tracker records user-initiated activities that change the state of a service in IBM Cloud. You can use this service to investigate abnormal activity and critical actions and to comply with regulatory audit requirements. In addition, you can be alerted about actions as they happen. The events that are collected comply with the Cloud Auditing Data Federation (CADF) standard. For more information, see the `getting started tutorial for IBM Cloud Activity Tracker <https://test.cloud.ibm.com/docs/activity-tracker?topic=activity-tracker-getting-started>`__.
```
Comment (jyu00) [inline docs/cloud/at-events.rst:6]: The link points to `test.cloud.ibm.com`

File: docs/cloud/at-events.rst [hunk 12]
```diff
@@ -0,0 +1,38 @@
...
++--------------------------------------+-------------------------------------------------------------------------------------------------------------------------------------------------+
+| ``quantum-computing.job.cancel``     | An event is generated when a job is cancelled.                                                                                                  |
++--------------------------------------+-------------------------------------------------------------------------------------------------------------------------------------------------+
+
+Viewing events
+--------------
+
+Events that are generated by an instance of the *YourServiceName* service are automatically forwarded to the IBM Cloud Activity Trackerservice instance that is available in the same location.
+
+IBM Cloud Activity Tracker can have only one instance per location. To view events, you must access the web UI of the IBM Cloud Activity Tracker service in the same location where your service instance is available. For more information, see `Launching the UI <https://test.cloud.ibm.com/docs/activity-tracker?topic=activity-tracker-launch>`__.
```
Comment (jyu00) [inline docs/cloud/at-events.rst:38]: This one also points to `test.cloud.ibm.com`

File: docs/cloud/release-notes.rst [hunk 13]
```diff
@@ -0,0 +1,33 @@
+Release notes for Qiskit Runtime
```
Comment (jyu00) [inline docs/cloud/release-notes.rst:1]: We probably don't need this anymore right? Now that we have the `Release Notes` section for actual qiskit-ibm-runtime releases.

File: docs/how_to/error-mitigation.rst (hunk 1 above)
Comment (beckykd) [reply docs/how_to/error-mitigation.rst:147]: Fair - I thought that was confusing too but forgot to ask!

File: docs/cloud/data-security.rst (hunk 9 above)
Comment (beckykd) [reply docs/cloud/data-security.rst:16]: They're in md instead of rst.

File: docs/cloud/release-notes.rst (hunk 13 above)
Comment (beckykd) [reply docs/cloud/release-notes.rst:1]: Makes sense to me

**Pull request merged**